amp_conf/bin/module_admin

```diff
@@ -379,16 +379,19 @@ function showi18n($modulename) {
 	$settingsProcessed = array();
 	foreach ($conf as $keyword => $settings) {
 		//we don't need hidden settings as the user never sees them
-		if ($settings['hidden'] != true && ($rawname == 'framework' && $settings['module'] == '') || $settings['module'] == $modulename) {
-			if (!in_array($settings['name'], $settingsProcessed['name'])) {
+		if ($settings['hidden'] == true) {
+			continue;
+		} elseif (($rawname == 'framework' && $settings['module'] == '') ||
+			$settings['module'] == $modulename) {
+			if (!in_array($settings['name'], $settingsProcessed)) {
```

Comment:
This looks broken (as do the other three). We are inserting into ['name'], but you are checking higher up. in_array is not recursive.